functional-tests/pages/additionalParams.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { BasePage } = require('kakunin');
+
+class AdditonalParamsPage extends BasePage {
+  constructor() {
+    super();
+
+    this.url = '/navigation/pages/myPageId/titles/myPageTitle?additionalParam1=value1&additionalParam2=value2';
```

Comment:
Let's add a test where we got a `this.url = /navigation/pages/:id/titles/myPageTitle?additionalParam1=value1&additionalParam2=value2`

where the ID just a random page number, so we're sure that the whole functionality works as expected.

Reply:
test updated